include/fem-dem/parameters_cfd_dem.h

```diff
@@ -44,7 +44,8 @@ namespace Parameters
   {
     function,
     pcm, // The particle centered method
-    qcm  // The quadratured centered method
+    qcm, // The quadratured centered method
```

Comment:
If you decide to change the name of the parameters, don't forget to do it here also.